Tests/ConnectionFactoryTest.php

```diff
@@ -51,7 +54,7 @@ public function testContainer(): void
         try {
             $factory->createConnection($params, $config, $eventManager, $mappingTypes);
         } catch (Throwable $e) {
-            $this->assertTrue(strpos($e->getMessage(), 'can circumvent this by setting') > 0);
+            $this->assertStringContainsString($e->getMessage(), 'can circumvent this by setting');
```

Comment:
It should be the other way around:

```suggestion
            $this->assertStringContainsString('can circumvent this by setting', $e->getMessage());
```